### DISCUSSION TITLE
Delayed - still here - not leaving 😄

### DISCUSSION THREAD

**Discussion by TonyGravagno** [araera111/mysql-to-zod#69] [General]
I have not been monitoring GitHub for a few days. But I have read some notes here and will catch up soon.

I have been modifying my refactoring code that uses the code generated from this utility as source, to generate modules for my application. So I have been working with this, just not "here". :)

I will apologize for recent delays, but apologies aren't really necessary from anyone - this is FOSS and we do what we can when we can.

Thank YOU, Ou Fumio, for your enthusiasm and effort on this wonderful code!

**Comment by araera111** [comment]
Thanks for your help.

I will be moving on September 8, so I will be gone for a few days before and after that.
Thanks to you, I enjoy coding every day.

**Comment by TonyGravagno** [comment]
Hello my friend! I had to shift projects for a while but I will be coming back to the Zod code soon. I miss our productive exchanges. I look forward to seeing all of the great things you have done here, and to continue working with you.

**Comment by araera111** [comment]
I have successfully moved into a house and live with my two little cats and one big wife.

Recently I have been working on a different project than mysql-to-zod.

As far as mysql-to-zod is concerned, I have created a sync option, and I have created English and Japanese documents. If you have time, I would greatly appreciate it if you could point out the unnatural parts in the English document.
[https://mysql-to-zod.pages.dev/](https://mysql-to-zod.pages.dev/)

**Reply by TonyGravagno** [reply]
It took a long time but I'm back to working on my schema code. Thank you for accepting my contributions to your documentation. I like the Astro docs and that you published to pages.dev. I don't fully understand that software or the Cloudflare integration but apparently I don't need to know how it works to contribute. Good choices!! Thanks!
